components/LogoNav/LogoNav.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import LogoNav from './LogoNav';
+
+describe('LogoNav Component', () => {
+  it('renders the logo and links to the /league/all page', () => {
+    render(<LogoNav />);
+
+    const logoImage = screen.getByTestId('logo-nav');
+    expect(logoImage).toBeInTheDocument();
+
+    const linkElement = screen.getByRole('link', {
+      name: /Gridiron Survivor logo/i,
```

Comment:
General question, what's the forward-slashes and "i" do here?

Reply:
@ryandotfurrer it's regex to look for the specific word of 'Gridiron Survivor Logo'. Here's a great blog about the i in Regex: https://www.w3schools.com/jsref/jsref_regexp_i.asp. Please let me know if this makes sense :)